component/sections/portfolio/variant_b.js

```diff
@@ -1,182 +1,95 @@
+  };
+
   return (
     <section>
       <div className="skew skew-top mr-for-radius">
-        <svg
-          className="h-8 md:h-12 lg:h-20 w-full text-gray-50"
-          viewBox="0 0 10 10"
-          preserveAspectRatio="none"
-        >
+        <svg className="h-8 md:h-12 lg:h-20 w-full text-gray-50" viewBox="0 0 10 10" preserveAspectRatio="none">
           <polygon fill="currentColor" points="0 0 10 10 0 10" />
         </svg>
       </div>
       <div className="skew skew-top ml-for-radius">
-        <svg
-          className="h-8 md:h-12 lg:h-20 w-full text-gray-50"
-          viewBox="0 0 10 10"
-          preserveAspectRatio="none"
-        >
+        <svg className="h-8 md:h-12 lg:h-20 w-full text-gray-50" viewBox="0 0 10 10" preserveAspectRatio="none">
           <polygon fill="currentColor" points="0 10 10 0 10 10" />
         </svg>
       </div>
       <div className="py-20 bg-gray-50 radius-for-skewed">
         <div className="container px-4 mx-auto">
           <div className="mb-16 flex flex-wrap justify-center md:justify-between items-center">
             <div className="text-center lg:text-left">
-              {caption && (
-                <span className="text-webriq-darkblue font-bold">
-                  {caption}
-                </span>
-              )}
-              {title && (
-                <h2 className="text-4xl lg:text-5xl font-bold font-heading">
-                  {title}
-                </h2>
-              )}
+              {caption && <span className="text-webriq-blue font-bold">{caption}</span>}
+              {title && <h2 className="text-4xl lg:text-5xl font-bold font-heading">{title}</h2>}
             </div>
-            {primaryButton?.label && (
-              <a
-                className="md:inline-block py-2 px-6 rounded-l-xl rounded-t-xl bg-webriq-blue hover:bg-webriq-darkblue text-gray-50 font-bold leading-loose transition duration-200"
-                target={primaryButton?.linkTarget}
-                rel={
-                  primaryButton?.linkTarget === "_blank"
-                    ? "noopener noreferrer"
-                    : null
-                }
-                href={
-                  primaryButton.type === "linkInternal"
-                    ? primaryButton.internalLink === "Home" ||
-                      primaryButton.internalLink === "home"
-                      ? "/"
-                      : primaryButton.internalLink
-                    : primaryButton.externalLink
-                }
+            {portfolios.length === viewProjects || portfolios.length < 6 ? null : (
+              <button 
+                className="hidden md:inline-block py-2 px-6 rounded-l-xl rounded-t-xl bg-webriq-blue hover:bg-webriq-darkblue text-gray-50 font-bold leading-loose transition duration-200" 
+                onClick={handleViewMoreProjects}
               >
-                {primaryButton.label}
-              </a>
+                View More Projects
```

Comment:
Is this intentional? We've gone from dynamic label to a static text `View More Projects`

Reply:
No, I missed to update this one including its corresponding schema field. Revised code including [here](https://github.com/webriq-pagebuilder/sanity-packages/pull/10). Thank you!

Reply:
Awesome!